extension/scripts/features/job-specials/ttJobSpecials.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+(async () => {
+	if (!getPageStatus().access) return;
+
+	featureManager.registerFeature(
+		"Job Specials",
+		"joblist",
+		() => settings.pages.joblist.specials,
+		null,
+		showSpecials,
+		removeSpecials,
+		{
+			storage: ["settings.pages.joblist.specials"],
+		},
+		null
+	);
+
+	async function showSpecials() {
+		await requireElement(".content-wrapper .company-details");
+
+		const { content } = createContainer("Job Specials", {
+			previousElement: document.find(".company-details-wrap"),
+			spacer: true,
+		});
+
+		const companyType = document.find(".details-wrap ul.info .m-title .m-show:not(.arrow-left)").innerText.trim();
+
+		for (const stars of [1, 3, 5, 7, 10]) {
+			const companyInfo = COMPANY_INFORMATION[companyType];
+			const name = companyInfo[stars].name;
+			const cost = companyInfo[stars].cost;
+			const effect = companyInfo[stars].effect;
+
+			content.appendChild(document.newElement({
+				type: "div",
+				class: `tt-comp-info-wrap ${await checkMobile() ? "mobile" : ""}`,
```

Comment:
Do the mobile await as a requirement and use `mobile` in the feature itself.

Reply:
Actually, it's not even needed.

1. You don't use it anywhere.
2. If it's just for styling, `body.tt-mobile` will do it.